js-release/src/index.ts

```diff
@@ -0,0 +1,127 @@
+import { exec } from "child_process";
+import readdirp from "readdirp";
+import * as core from "@actions/core";
+import { Storage } from "@google-cloud/storage";
+import setupGCloud from "./setup-gcloud/setup-gcloud";
+
+// config
+const BUCKET = "manifold-js";
+const URL_MAP = "manifold-js-cdn-urlmap";
```

Comment:
Minor/optional: I know we aren't planning on this - but you could make BUCKET and URL_MAP as parameters which have these as default values here if you like and make the action more configurable

Reply:
I considered that, but I’m on the fence. These specific naming & URL Map rules only apply to JS projects, so I don’t think this value would ever change. And I don’t want to allow repos the ability to mess something up through a misconfiguration

Reply:
I don’t think we’d need to configure this, but we could always release it to the GitHub marketplace if we wanted to—if other people wanted to recreate unpkg.com on their own GCP instance, which other people might want